src/core.jsx

```diff
@@ -91,7 +92,13 @@ export default (
 
     render() {
       if (!this.isLoaded()) {
-        return <LoadingIndicator {...this.state.props} />
+        if (LoadingIndicator) {
```

Comment:
We want to avoid multiple inputs.
The idea behind the `isFunction` is to have only one `LoadingIndicator` parameter.

So, what we could do is : 
```es6
if (!LoadingIndicator) return null
if (isFunction(LoadingIndicator)) return LoadingIndicator(this.state.props)
return <LoadingIndicator className={this.state.props.className} style={this.state.props.style}
```